### ISSUE TITLE
ERROR: Service 'php-worker' failed to build

### ISSUE THREAD

**Issue by andrew-narolsky** [laradock/laradock#3013]
When i use LARADOCK_PHP_VERSION=8.0 i get error 
...
(70/76) Installing py3-six (1.15.0-r1)
(71/76) Installing py3-packaging (20.9-r1)
(72/76) Installing py3-setuptools (52.0.0-r3)
(73/76) Installing supervisor (4.2.2-r2)
(74/76) Installing libunistring (0.9.10-r1)
(75/76) Installing libidn2 (2.3.1-r0)
(76/76) Installing wget (1.21.1-r1)
Executing busybox-1.33.1-r2.trigger
1 error; 363 MiB in 108 packages
Removing intermediate container 6198345a6890
The command '/bin/sh -c apk --update add wget   curl   git   build-base   libmemcached-dev   libmcrypt-dev   libxml2-dev   pcre-dev   zlib-dev   autoconf   cyrus-sasl-dev   libgsasl-dev   oniguruma-dev   libressl   libressl-dev   supervisor' returned a non-zero code: 1
ERROR: Service 'php-worker' failed to build

**Comment by andrew-narolsky** [comment]
...
(70/76) Installing py3-six (1.15.0-r1)
(71/76) Installing py3-packaging (20.9-r1)
(72/76) Installing py3-setuptools (52.0.0-r3)
(73/76) Installing supervisor (4.2.2-r2)
(74/76) Installing libunistring (0.9.10-r1)
(75/76) Installing libidn2 (2.3.1-r0)
(76/76) Installing wget (1.21.1-r1)
Executing busybox-1.33.1-r2.trigger
1 error; 363 MiB in 108 packages
Removing intermediate container 6198345a6890
The command '/bin/sh -c apk --update add wget   curl   git   build-base   libmemcached-dev   libmcrypt-dev   libxml2-dev   pcre-dev   zlib-dev   autoconf   cyrus-sasl-dev   libgsasl-dev   oniguruma-dev   libressl   libressl-dev   supervisor' returned a non-zero code: 1
ERROR: Service 'php-worker' failed to build

**Comment by garbinmarcelo** [comment]
Hi, did you solve it? I'm getting the same error with PHP_VERSION=7.4

**Comment by harmakit** [comment]
Same
LARADOCK_PHP_VERSION=8.0

```
Building php-worker
Step 1/76 : ARG LARADOCK_PHP_VERSION
Step 2/76 : FROM php:${LARADOCK_PHP_VERSION}-alpine
 ---> b224257ffa36
Step 3/76 : LABEL maintainer="Mahmoud Zalt <mahmoud@zalt.me>"
 ---> Using cache
 ---> d408e3470cb3
Step 4/76 : ARG LARADOCK_PHP_VERSION
 ---> Using cache
 ---> be51b94685fc
Step 5/76 : ARG CHANGE_SOURCE=false
 ---> Using cache
 ---> ea0dcca4c4e5
Step 6/76 : RUN if [ ${CHANGE_SOURCE} = true ]; then     sed -i 's/dl-cdn.alpinelinux.org/mirrors.aliyun.com/' /etc/apk/repositories ;fi
 ---> Using cache
 ---> 1c0ea5d79347
Step 7/76 : RUN apk --update add wget   curl   git   build-base   libmcrypt-dev   libxml2-dev   pcre-dev   zlib-dev   autoconf   cyrus-sasl-dev   libgsasl-dev   oniguruma-dev   libressl   libressl-dev   supervisor
 ---> Running in e303ee29aaa3
fetch https://dl-cdn.alpinelinux.org/alpine/v3.14/main/x86_64/APKINDEX.tar.gz
fetch https://dl-cdn.alpinelinux.org/alpine/v3.14/community/x86_64/APKINDEX.tar.gz
(1/74) Installing m4 (1.4.18-r2)
(2/74) Installing libbz2 (1.0.8-r1)
(3/74) Installing perl (5.32.1-r0)
(4/74) Installing autoconf (2.71-r0)
(5/74) Installing libgcc (10.3.1_git20210424-r2)
(6/74) Installing libstdc++ (10.3.1_git20210424-r2)
(7/74) Installing binutils (2.35.2-r2)
(8/74) Installing libmagic (5.40-r1)
(9/74) Installing file (5.40-r1)
(10/74) Installing libgomp (10.3.1_git20210424-r2)
(11/74) Installing libatomic (10.3.1_git20210424-r2)
(12/74) Installing libgphobos (10.3.1_git20210424-r2)
(13/74) Installing gmp (6.2.1-r0)
(14/74) Installing isl22 (0.22-r0)
(15/74) Installing mpfr4 (4.1.0-r0)
(16/74) Installing mpc1 (1.2.1-r0)
(17/74) Installing gcc (10.3.1_git20210424-r2)
(18/74) Installing musl-dev (1.2.2-r3)
(19/74) Installing libc-dev (0.7.2-r3)
(20/74) Installing g++ (10.3.1_git20210424-r2)
(21/74) Installing make (4.3-r0)
(22/74) Installing fortify-headers (1.1-r1)
(23/74) Installing patch (2.7.6-r7)
(24/74) Installing build-base (0.5-r2)
(25/74) Installing gdbm (1.19-r0)
(26/74) Installing libsasl (2.1.27-r12)
(27/74) Installing pkgconf (1.7.4-r0)
(28/74) Installing cyrus-sasl-dev (2.1.27-r12)
(29/74) Installing expat (2.4.1-r0)
(30/74) Installing pcre2 (10.36-r0)
(31/74) Installing git (2.32.0-r0)
(32/74) Installing perl-error (0.17029-r1)
(33/74) Installing perl-git (2.32.0-r0)
(34/74) Installing git-perl (2.32.0-r0)
(35/74) Installing libgpg-error (1.42-r0)
(36/74) Installing libgcrypt (1.9.3-r0)
(37/74) Installing krb5-conf (1.0-r2)
(38/74) Installing libcom_err (1.46.2-r0)
(39/74) Installing keyutils-libs (1.6.3-r0)
(40/74) Installing libverto (0.3.2-r0)
(41/74) Installing krb5-libs (1.18.3-r1)
(42/74) Installing libidn (1.37-r0)
(43/74) Installing libgsasl (1.8.1-r2)
(44/74) Installing libgsasl-dev (1.8.1-r2)
(45/74) Installing libmcrypt (2.5.8-r9)
(46/74) Installing libmcrypt-dev (2.5.8-r9)
(47/74) Installing libressl3.3-libcrypto (3.3.3-r0)
(48/74) Installing libressl3.3-libssl (3.3.3-r0)
(49/74) Installing libressl3.3-libtls (3.3.3-r0)
ERROR: libressl3.3-libtls-3.3.3-r0: trying to overwrite usr/lib/libtls.so.20 owned by libretls-3.3.3-r0.
ERROR: libressl3.3-libtls-3.3.3-r0: trying to overwrite usr/lib/libtls.so.20.0.3 owned by libretls-3.3.3-r0.
(50/74) Installing libressl (3.3.3-r0)
(51/74) Installing libressl-dev (3.3.3-r0)
(52/74) Installing zlib-dev (1.2.11-r3)
(53/74) Installing xz-dev (5.2.5-r0)
(54/74) Installing libxml2-dev (2.9.12-r1)
(55/74) Installing oniguruma-dev (6.9.7.1-r0)
(56/74) Installing libpcre16 (8.44-r0)
(57/74) Installing libpcre32 (8.44-r0)
(58/74) Installing pcre (8.44-r0)
(59/74) Installing libpcrecpp (8.44-r0)
(60/74) Installing pcre-dev (8.44-r0)
(61/74) Installing libffi (3.3-r2)
(62/74) Installing mpdecimal (2.5.1-r1)
(63/74) Installing readline (8.1.0-r0)
(64/74) Installing python3 (3.9.5-r1)
(65/74) Installing py3-ordered-set (4.0.2-r1)
(66/74) Installing py3-appdirs (1.4.4-r2)
(67/74) Installing py3-parsing (2.4.7-r2)
(68/74) Installing py3-six (1.15.0-r1)
(69/74) Installing py3-packaging (20.9-r1)
(70/74) Installing py3-setuptools (52.0.0-r3)
(71/74) Installing supervisor (4.2.2-r2)
(72/74) Installing libunistring (0.9.10-r1)
(73/74) Installing libidn2 (2.3.1-r0)
(74/74) Installing wget (1.21.1-r1)
Executing busybox-1.33.1-r2.trigger
1 error; 363 MiB in 106 packages
ERROR: Service 'php-worker' failed to build: The command '/bin/sh -c apk --update add wget   curl   git   build-base   libmcrypt-dev   libxml2-dev   pcre-dev   zlib-dev   autoconf   cyrus-sasl-dev   libgsasl-dev   oniguruma-dev   libressl   libressl-dev   supervisor' returned a non-zero code: 1
```

**Comment by andrew-narolsky** [comment]
change php-worker's dockerfile FROM php:${LARADOCK_PHP_VERSION}-alpine3.13

**Comment by da-mask** [comment]
I had this same issue, @andrew-narolsky 's solution solved it.  Following for a permanent fix.

**Comment by garbinmarcelo** [comment]
Hi, has a pull request been opened to fix this?

**Comment by lionslair** [comment]
> change php-worker's dockerfile
> FROM php:${LARADOCK_PHP_VERSION}-alpine3.13

FYI same fix as here https://github.com/laradock/laradock/issues/3008

**Comment by da-mask** [comment]
I think this should be closed now, the release of Alpine 3.14.1 should have fixed this.
